### ISSUE TITLE
Chapter 5 issues

### ISSUE THREAD

**Issue by jsmestad** [balinterdi/rarwe-issues#11]
Issues with Chapter 5 are below. It seems up until Chapter 5, things were well explained step-by-step. Once getting to Chapter 5 things seem to be a bit chaotic where `Band.create({name: '....', songs: [....]})` shows up on page 59, but is not even mentioned to exist in the text until page 64.

### Issues

In Chapter 5, it appears you all of a sudden stop calling out the module syntax without warning. You warned in Chapter 1 that you'd drop the use of `import Ember from 'ember';` but in Chapter 5 the syntax of `export default Router.map(function() {` is dropped to just `Router.map(function() {`.

Can you please either update the code snippets or call out that the module export is implied?

----

it says to `delete app/routes/songs.js` early in the chapter, than later says to copy content over from that. - Additional change was you assigned Pearl Jam's 'Yellow Ledbetter' to a variable called `yellowLedbetter` in Chapters 1 to 4, in Chapter 5 when you copy it over from songs.js you've changed the variable name to `ledbetter`

----

Page 60 it says:

Here, we passed `bands` to modelFor to access all the bands that had previously been fetched by the `band` route. 

It seems like it should say:

Here, we passed `bands` to modelFor to access all the bands that had previously been fetched by the `bands` route. 

----

Page 65 in Nested Templates there is a few issues with the sentence:

We saw in the [Routing chapter]() that templates must be named to exactly match as their corresponding routes. 

Remove the word `as` from `match as`

----

Page 66

    We also want to change how songs are displayed. They are now displayed on their own page, but we want to show them next to the list of bands and mark the band to which they belong, just as the comp showed at the beginning of the chapter:

You had us delete all of this functionality as the first thing in Chapter 5. Meaning that "They are now displayed on their own page" is incorrect. We currently do not have songs displayed anywhere as those files were removed.

**Comment by jsmestad** [comment]
Additional issue on Page 64. 

After adding:

```javascript
var Band = Ember.Object.extend({ 
  name: '',

  (...)

  songs: function() {
    return songs.filterBy('band', this.get('name'));
  }.property('name', 'songs.@each.band')
});
```

There is a new JSHint Error in the ember-cli server output (pointing to the `return songs.filterBy('band', this.get('name'));` line): 

```
routes/bands.js: line 41, col 12, 'songs' is not defined.
```

**Comment by jsmestad** [comment]
Additional Issue. The screenshot on Page 67 is wrong. The box on the right does not look like the one included in the page.
![screen shot 2015-04-21 at 2 08 18 pm](https://cloud.githubusercontent.com/assets/16150/7261559/02e401f0-e830-11e4-8638-4705af168743.png)

**Comment by jsmestad** [comment]
No mention in Chapter 5 about what to do with this code or if its not needed anymore.

```javascript
var SongCollection = Ember.ArrayProxy.extend(Ember.SortableMixin, {
  sortProperties: ['rating'],
  sortAscending: false,
  content: []
});
```

You don't notice its gone due to the explicit order you specify the songs in:

```javascript
var pearlJam = Band.create({name: 'Pearl Jam', songs: [ledbetter, daughter]});
```

----

If you come back to it later, it'd be nice to mention that sorting will come back in later chapters but we lose it for now.

**Comment by jsmestad** [comment]
@balinterdi at the end of Chapter 5, should the root URL not display anything? I get proper content by manually going to `/bands` but before that I dont get anything. See attached.
![screen shot 2015-04-24 at 9 49 55 am](https://cloud.githubusercontent.com/assets/16150/7322761/60c51fdc-ea67-11e4-8359-b7480d54f2b6.png)

**Comment by balinterdi** [comment]
That's right, one should go to `/bands` until a few chapters later. I've added a paragraph to make this clear.

**Comment by balinterdi** [comment]
Fixed for 1.11.1
